### PR TITLE
allow zmon to access given buckets

### PR DESCRIPTION
**Pull request by vetinari**
so we do not override what users might have set

### CODE REVIEW

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -703,3 +703,6 @@ deployment_service_enabled: "true"
 
 # Standard storageclass gp3 volume type
 storageclass_standard_gp3: "false"
+
+# default for ZMON is not to be able to get any objects from S3
+zmon_s3_allow_get_object: ""
```
Comment (linki) [inline cluster/config-defaults.yaml]: How about we make the default "false" similar to the `storageclass_standard_gp3` setting above.

Comment (linki) [reply cluster/config-defaults.yaml]: Nevermind, I see now that it's a list of buckets and not a bool.

Comment (linki) [reply cluster/config-defaults.yaml]: Then let's rename the field to something like: `zmon_accessible_s3_buckets` and add add a comment that states that it's a list of buckets separated by comma.

**Comment by linki** [comment]
Where can users set this?

**Comment by mikkeloscar** [comment]
> Where can users set this?

I guess they would have to request this from the ZMON team. Right @vetinari?

**Comment by mikkeloscar** [comment]
:+1:

**Comment by vetinari** [comment]
> I guess they would have to request this from the ZMON team. Right @vetinari?

yes

**Comment by vetinari** [comment]
:+1:

**Comment by linki** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**